src/main/flight/pid.c

```diff
@@ -910,7 +910,7 @@ NOINLINE static void applySpa(int axis, const pidProfile_t *pidProfile)
 #else
     UNUSED(axis);
     UNUSED(pidProfile);
-#endif // #ifdef USE_WING ... #else
+#endif // USE_WING ... #else
```

Comment:
```suggestion
#endif // USE_WING
```

Reply:
well, it has #else :) so not a regular #endif

Reply:
#else is close here, it is obvious here.
In general, simple USE_WING is usually enough. IMO normal way it to look at directive before code, so #endif just signifies block ends and  `// USE_WING` is hint when searching for beginning of block.